crates/cosmos/cosmos-chain-components/src/impls/transaction/poll_timeout.rs

```diff
@@ -5,15 +5,11 @@ use hermes_relayer_components::transaction::impls::poll_tx_response::{
     PollTimeoutGetter, PollTimeoutGetterComponent,
 };
 
-pub struct DefaultPollTimeout;
+pub struct FixedPollTimeoutMillis<const MILLIS: u64>;
```

Comment:
Here should be seconds, or that we would need to use `FixedPollTimeoutMillis<300 * 60>` or else the timeout is too short.

Reply:
Fixed this by using seconds instead of milliseconds 4881c92cf380432d546ba3869b875937a921c104